### ISSUE TITLE
Important Note for Hyper-V isloation:

### ISSUE THREAD

**Issue by msm-fc** [MicrosoftDocs/Virtualization-Documentation#888]
Please mention that if Hyper-V isolation is enabled in Windows 10, other hypervisors such as VirtualBox may fail because hyper-v disables vt-x for other hypervisors running on the same hardware.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 66894009-bd4f-3fa6-6e03-a8485686bea5
* Version Independent ID: 9d081837-7987-eeb5-1abf-37ebc2e3c7f6
* Content: [Windows Container on Windows 10](https://docs.microsoft.com/en-us/virtualization/windowscontainers/quick-start/quick-start-windows-10)
* Content Source: [virtualization/windowscontainers/quick-start/quick-start-windows-10.md](https://github.com/Microsoft/Virtualization-Documentation/blob/live/virtualization/windowscontainers/quick-start/quick-start-windows-10.md)
* Service: **windows-containers**
* Product: **windows-containers**
* GitHub Login: @taylorb-microsoft

**Comment by daniel-kun** [comment]
Yes, this is a major headache. Docker for Windows and VirtualBox can't be used on the same machine without a reboot ;-(

**Comment by robinharwood** [comment]
We hugely appreciate your contribution, feedback, and support to our technical documentation.

We’ve phased out GitHub Issues on this repository as the feedback mechanism for content and have replaced it with a new feedback system. For more information see, [Content User Feedback](https://aka.ms/ContentUserFeedback). As part of this rollout, we are migrating your issue where our team will use your comments to improve the documentation experience. Unfortunately, this does mean you won't be able to access this issue after the migration is complete.

All issues will be migrated by Thursday 29th August 2024.

Once again, thank you for your valuable feedback, we hope you’ll continue to contribute using our new feedback system.
